src/main/java/ithillel/bank/tables/account/AccountDto.java

```diff
@@ -7,6 +7,7 @@ public record AccountDto(
         String uid,
         String iban,
         double balance,
-        String personUid
+        String personUid,
```

Comment:
```suggestion
        String personId,
```
краще так, якщо нагадаєш на уроці - поговоримо чому.